examples/bpk-component-overlay/examples.js

```diff
@@ -34,140 +33,145 @@ const getClassName = cssModules(STYLES);
 
 const SolidExamples = () => (
   <div className={getClassName('bpk-overlay-stories')}>
-    {Object.keys(OVERLAY_LEVELS).map((level) => {
-      if (level !== OVERLAY_LEVELS.off) {
-        return (
-          <div className={getClassName('bpk-overlay-stories__overlay-story')}>
-            <BpkOverlay overlayType={OVERLAY_TYPES.solid} overlayLevel={level}>
-              <BpkImage
-                src={IMAGE_SRC}
-                altText="Ocean Ripple by Matt Hardy"
-                aspectRatio={1.8}
-              />
-            </BpkOverlay>
-            <div className={getClassName('bpk-overlay-stories__overlay--name')}>
-              <BpkText textStyle={TEXT_STYLES.xl}>
-                Solid {level.charAt(0).toUpperCase() + level.slice(1)}
-              </BpkText>
-            </div>
-          </div>
-        );
-      }
-      return null;
-    })}
+    {[
+      OVERLAY_TYPES.solidLow,
+      OVERLAY_TYPES.solidMedium,
+      OVERLAY_TYPES.solidHigh,
+    ].map((overlayType) => (
+      <div
+        key={overlayType}
+        className={getClassName('bpk-overlay-stories__overlay-story')}
+      >
+        <BpkOverlay overlayType={overlayType}>
+          <BpkImage
+            src={IMAGE_SRC}
+            altText="Ocean Ripple by Matt Hardy"
+            aspectRatio={1.8}
+          />
+        </BpkOverlay>
+        <div className={getClassName('bpk-overlay-stories__overlay--name')}>
+          <BpkText textStyle={TEXT_STYLES.xl}>
+            {overlayType.charAt(0).toUpperCase() + overlayType.slice(1)}
+          </BpkText>
```

Comment:
Couple suggestions for improving the docs:
- move this to a separate component to reused across all examples:
```suggestion
          <OverlayName overlayType={overlayType} />
```

```tsx
const OverlayName = ({ overlayType }) => (
  <BpkText textStyle={TEXT_STYLES.xl}>
    {overlayType.charAt(0).toUpperCase() + overlayType.slice(1)}
  </BpkText>)
```
- if you add space, there should be no need in re-generating new storybook screenshot, they should match – easier for review:
```suggestion
          <BpkText textStyle={TEXT_STYLES.xl}>
            {`${overlayType.charAt(0).toUpperCase()}  ${overlayType.slice(1)}`}
          </BpkText>
```
- or instead of all of this, just use `overlayType`, as the doc is the for web developers, so it would be easier for devs to just see the name and copy-paste it into their codebase. Small thing, but 🤷 
```suggestion
          <BpkText textStyle={TEXT_STYLES.xl}>
            {overlayType}
          </BpkText>
```